src/cmd/lint.js

```diff
@@ -5,11 +5,59 @@ import {FileFilter} from './build';
 
 const log = createLogger(__filename);
 
+// Flow types
+
+export type LinterLogLevel = 'debug' | 'fatal';
+export type LinterOutputType = 'text' | 'json';
+
+export type LinterCreatorParams = {
+  config: {
+    logLevel: LinterLogLevel,
```

Comment:
Should this just be `logLevel: 'debug' | 'fatal'`? I don't see `LinterLogLevel` referenced elsewhere.